include/circt/Dialect/Handshake/Simulation.h

```diff
@@ -17,6 +17,11 @@
 
 // Functions used to execute a restricted form of the standard dialect, and
 // the handshake dialect.
+#include <stdio.h>
+
+using namespace llvm;
+using namespace mlir;
+using cl::opt;
 
 bool simulate(llvm::cl::opt<std::string> &toplevelFunction,
```

Comment:
Unrelated to your patch, but should/can `simulate` take a `StringRef toplevelFunction, ArrayRef<std::string> inputArgs` instead of taking cl::opt's?  The fact that these come from cl::ops in the driver is an implementation detail.